RELEASE_NOTES.md

```diff
@@ -7,51 +7,46 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# knife-azure 1.7.0 release notes:
-In this release, few options have been removed since their support has been removed from the azure-chef-extension. The options are: `--auto-update-client`, `--uninstall-chef-client` and `--delete-chef-extension-config`. They were earlier supported with `cloud-api` bootstrap protocol which uses azure-chef-extension for installing chef-client.
+# knife-azure 1.8.0 release notes:
+In this release added 'daemon' option and renamed`--chef-service-interval` option to `--chef-daemon-interval`.
+
+Updated code to work with latest azure-sdk gems i.e. version 0.9.0.
 
 New options introduced:
-`--chef-service-interval`, which lets the user specify the frequency at which chef-client service should run. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
 
-`--extended-logs` shows chef convergence logs in detail. This option is available for both ASM and ARM commands. For ASM, it's supported with `cloud-api` bootstrap protocol.
+`--azure_availability_set`, which allows user to create virtual machines in required availability set.
+
+`--daemon`, which allows user to select options to run chef-client as none, service or task. This option works for windows node and --bootstrap-protocol to be 'cloud-api'.
```

Comment:
> which allows user to select options to run chef-client as none, service or task.

which allows the user to configure the chef-client to run as a daemon by specifying `none`, `service`, or `task`.

> This option works for windows node and --bootstrap-protocol to be 'cloud-api'.

I don't understand this. Do we mean to say "This option is only supported on Windows and requires the `--bootstrap-protocol` option to be set to `cloud-api`."